lib/appmap.rb

```diff
@@ -112,17 +112,19 @@ def detect_metadata
   end
 end
 
-if defined?(::Rails::Railtie)
```

Comment:
Due to load order, `::Rails::Railtie` may well not be defined here; even if Rails is in the bundle.